modules/virt-configuring-huge-pages-for-vms.adoc

```diff
@@ -22,7 +22,7 @@ If you edit a running virtual machine, the virtual machine must be rebooted for
 
 .Prerequisites
 
-* Nodes must have pre-allocated huge pages configured. For instructions, see link:https://docs.openshift.com/container-platform/{product-version}/scalability_and_performance/what-huge-pages-do-and-how-they-are-consumed-by-apps.html#configuring-huge-pages_huge-pages[Configuring huge pages at boot time].
+* Nodes must have pre-allocated huge pages configured. For instructions, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/latest/html/scalability_and_performance/what-huge-pages-do-and-how-they-are-consumed#configuring-huge-pages_huge-pages[Configuring huge pages at boot time].
```

Comment:
Should this have a link, where it is now in a module? 
```suggestion
* Nodes must have pre-allocated huge pages configured. For instructions, see Configuring huge pages at boot time.
```

Reply:
I thought so. This wasn't part of what I had to do for the Jira story, but the prow tests failed if didn't update this link, as it was to the  docs.openshift.com

Reply:
I believe that all content in the core OCP docs, anything in `main` or an `enterprise-4.*` branch is considered internal, thus cannot be linked to from a module.